fe/fe-core/src/test/java/com/starrocks/catalog/MaterializedViewTest.java

```diff
@@ -993,4 +996,38 @@ public void testCreateMVWithDuplicateIndexOrDuplicateColumn() throws Exception {
                 UserException.class,
                 () -> starRocksAssert.withMaterializedView(mvSql2));
     }
+
+    @Test
+    public void testBaseTableInfo(@Mocked GlobalStateMgr globalStateMgr,
```

Comment:
My question is why this UT is added to this file?
I think the UT should be added to [BaseTableInfo.java](https://github.com/StarRocks/starrocks/pull/34507/files#diff-735f3a6d5bd4e296e4d624e4dc3cac2ec744f48049f055ff56596cd483211632)

Reply:
this class is mv meta related, MaterializedViewTest is mv meta related test. So I add it here.

Reply:
BaseTableInfoTest.java is another choice, I move the test to it.